viz_scripts/scaffolding.py

```diff
@@ -254,7 +257,7 @@ async def map_trip_data(expanded_trip_df, study_type, dynamic_labels):
 
     return expanded_trip_df
 
-async def load_viz_notebook_inferred_data(year, month, program, study_type, dynamic_labels, include_test_users=False):
+async def load_viz_notebook_inferred_data(year, month, program, study_type, dynamic_labels, include_test_users=False, add_footprint=False):
```

Comment:
For my understanding, is this cleanup from something that was meant to be added in a previous PR?

I saw that `load_viz_notebook_data` already had the `add_footprint` parameter, but `load_viz_notebook_inferred_data` was missing it

Reply:
No, this is not a cleanup which was meant to be added in the previous PR. We were using `add_footprint` parameter to compute footprint calculations in `labeled trips` in `energy_calculations` notebook. Earlier, we were not using `add_footprint` parameter for `load_viz_notebook_inferred_data`.

We need to pass `add_footprint` parameter as `True` to enable extraction of `base_mode`. Therefore, this change is required, such that we have `base_mode` as column added in the `expanded_ct_inferred` data frame. This enables us for filtering AIR as the base_mode, instead of using mode_confirm_w_other = 'air'.